app/views/results/compare_view.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Comparison from <%= link_to(@result_left.id, "/results?id=#{@result_left.id}") %> to <%= link_to(@result_right.id, "/results?id=#{@result_right.id}") %></h1>
+
+<%= @results_comper.diffy.html_safe %>
```

Comment:
`h` is an alias for `html_escape` (http://api.rubyonrails.org/classes/ERB/Util.html#method-c-h); //not// `html_safe`